src/index.js

```diff
@@ -68,22 +67,11 @@ const ApifyClient = function (options = {}) {
         return (callOpts, callback) => {
             const mergedOpts = Object.assign({}, instanceOpts, callOpts);
 
+            // Remove traling forward slash from baseUrl.
+            if (mergedOpts.baseUrl.substr(-1) === '/') mergedOpts.baseUrl = mergedOpts.baseUrl.slice(0, -1);
```

Comment:
Shouldn't this be `instanceOpts.baseUrl` instead? and have this after the `if (!instanceOpts.baseUrl)` check?